bower.json

```diff
@@ -1,31 +1,28 @@
 {
-  "name": "purescript-html-parser-halogen",
-  "description": "A library to render HTML string into halogen views",
-  "keywords": [
-    "purescript",
-    "halogen",
-    "html"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/rnons/purescript-html-parser-halogen.git"
-  },
-  "license": "BSD-3-Clause",
-  "ignore": [
-    "**/.*",
-    "bower_components",
-    "docs",
-    "example",
-    "node_modules",
-    "output",
-    "yarn.lock"
```

Comment:
Thanks. I think these are still needed, so that `bower install` won't pull the example folder and yarn.lock.

It's good to keep the description field as well.

I don't know, maybe we can remove `"purescript-psci-support"`.